Makefile

```diff
@@ -110,6 +110,9 @@ bin/copy: $(SOURCES) tests/copy/copy.go
 bin/tutorial: $(SOURCES) tests/tutorial/tutorial.go
 	$(GO_BUILD) $(BUILDAH_LDFLAGS) -o $@ $(BUILDFLAGS) ./tests/tutorial/tutorial.go
 
+bin/inet: tests/inet/inet.go
```

Comment:
The scripts will need to be told where it is regardless, since it won't be in `../bin` relative to where the scripts end up packaged.  At that point it's the packager's decision, as it is with the other test helpers.